src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -886,6 +892,10 @@ case class BoolLit(b: Boolean)(val info: Source.Parser.Info) extends Lit {
   override def typ: Type = BoolT(Addressability.literal)
 }
 
+case class StringLit(s: String)(val info: Source.Parser.Info) extends Lit {
```

Comment:
Do you mean making `StringLit` a subtype of `StringOperation`? If so, I do not think that should be done because
(1) even though `StringLit` and `StringOperation` have the same type, they get their addressability from to different fields of the `Addressability` object: `literal` and `rValue`, respectively.
(2) for consistency: `IntLit` does not extend `IntOperation`. `BoolLit` also does not extend `BoolOperation`

Reply:
ok